src/OpenTelemetry/SuppressInstrumentationScope.cs

```diff
@@ -70,5 +43,10 @@ public void Dispose()
                 this.disposed = true;
             }
         }
+
+        internal static SuppressInstrumentationScope Begin(bool value = true)
```

Comment:
How do you feel about leaving this in?

`public static IDisposable Begin(bool value = true)`

Because I kind of like this way better:
`using (var scope = SuppressInstrumentationScope.Begin())`

But I'm OK also having the helper too 😁 

There's no more danger of `using (SuppressInstrumentationScope)` because the static is gone right?

Reply:
I feel great about this. I actually decided to remove the helper in favor of this idea. Though, happy to revert and put the helper back if others find it useful. I personally like your idea better.